### PR TITLE
Add release notes for 1.7.8.7

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add release notes for 1.7.8.7
| Fixed ticket? | N/A

### CODE REVIEW

File: news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+We would like to take the opportunity to stress out once more the importance of keeping your system updated to keep your shop safe from attacks. This means regularly updating both your PrestaShop software and its modules, as well as your server environment.
+
+## Acknowledgements
+
+Contributors to PrestaShop version 1.7.8.7:
+* Dominik Shaim who first reached out to the security team and helped investigate the vulnerability
+* Maintainers Daniel Hlavacek and Krystian Podemski who helped investigate the vulnerability
+* Maintainer and security lead Thomas Baccelli who helped investigate the vulnerability and build the fix
+* QA analysts and engineers who tested this release and the upgrade process: Florine Hea, Sarah Lorenzini, Khouloud Belguith, Hana Rebai, Marwa Chelly, Hibatallah Aouadni, Boubker Bribri and Aurelien Rita.
```
Comment (matks) [inline news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md]:
```suggestion
* QA analysts and engineers who tested this release and the upgrade process: Florine Hea, Sarah Cavrot, Khouloud Belguith, Hana Rebai, Marwa Chelly, Hibatallah Aouadni, Boubker Bribri and Aurelien Rita.
```
I think she changed her name 😅 https://github.com/slorenzini

File: news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ - 1.7.8.x
+ - 1.7
+
+---
+
+PrestaShop version 1.7.8.7 is now available. This release fixes one important security issue, read this article for more information.
+
+![1.7.8.7 is available!](/assets/images/2022/07/1.7.8.7_banner.jpg)
+
+The maintainer team has been made aware that malicious actors are exploiting a combination of known and unknown security security vulnerabilities to inject malicious code in PrestaShop websites, allowing them to execute arbitrary instructions, and potentially steal customer’s payment information. You can [read our dedicated article](https://build.prestashop.com/news/major-security-vulnerability-on-prestashop-websites/) to find out what’s happening, how the attack works and what to do to keep your shop safe.
```
Comment (coffrep) [inline news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md]:
```suggestion
The maintainer team has been made aware that malicious actors are exploiting a combination of known and unknown security vulnerabilities to inject malicious code in PrestaShop websites, allowing them to execute arbitrary instructions, and potentially steal customer’s payment information. You can [read our dedicated article](https://build.prestashop.com/news/major-security-vulnerability-on-prestashop-websites/) to find out what’s happening, how the attack works and what to do to keep your shop safe.
```

File: news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+While investigating this attack, we found a previously unknown vulnerability chain that has been fixed in this release. At the moment, however, we cannot be sure that it’s the only way for them to perform the attack.
+
+We suggest upgrading your shop quickly in order to benefit from this fix. Please remember to backup before you upgrade.
+
+Reminder: the latest version of the 1-Click Upgrade is v4.14.2 has been just released, don’t forget to upgrade it.
```
Comment (coffrep) [inline news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md]:
```suggestion
Reminder: the latest version of the 1-Click Upgrade is v4.14.2 and has been just released, don’t forget to upgrade it.
```

**Review by matks — APPROVED** [review]
OK for me

- Release on GitHub done
- Release on prestashop.com done
- Autoupgrade 4.14.2 done

Ready for publish for me 🎉

File: news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## About the security fix
+
+This security fix strengthens the MySQL Smarty cache storage against code injection attacks. Read the [security advisory here](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-hrgx-p36p-89q4).
+
+Note: this legacy feature will be removed from future PrestaShop versions and is maintained for backward compatibility reasons. It is recommended not to enable it due to reduced performance.
+
+We would like to take the opportunity to stress out once more the importance of keeping your system updated to keep your shop safe from attacks. This means regularly updating both your PrestaShop software and its modules, as well as your server environment.
```
Comment (coffrep) [inline news/_posts/releases/2022-07-25-prestashop-1-7-8-7-maintenance-release.md]:
```suggestion
We would like to take the opportunity to stress once more the importance of keeping your system updated to keep your shop safe from attacks. This means regularly updating both your PrestaShop software and its modules, as well as your server environment.
```

**Review by coffrep — DISMISSED** [review]

**Review by matks — APPROVED** [review]

**Comment by eternoendless** [comment]
Thanks for the review!

**Pull request merged**